### PR TITLE
add static function 'isRange' to moment

### PR DESCRIPTION
**Pull request by pablodgonzalez**
It's a little function, but useful to write more understandable code.
@gf3 I hope it coulbe useful to you too.
cheers!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
```
Comment (Rhysjc) [inline CHANGELOG.md]: Could you remove this change? We'll do it when we do a release

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,4832 @@
+{
```
Comment (Rhysjc) [inline package-lock.json]: Please remove this file and add it to .gitignore and .npmignore

File: webpack.config.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ module.exports = {
     filename: 'moment-range.js',
     library: 'moment-range',
     libraryTarget: 'umd',
-    path: './dist/',
+    path: __dirname + '/dist/',
```
Comment (Rhysjc) [inline webpack.config.js]: Please remove, I've seen there's some issues relating to this so we'll address it elsewhere 😄

File: CHANGELOG.md (hunk 1 above)
Comment (pablodgonzalez) [inline CHANGELOG.md]: do you want remove only this line --> ### Changed Ok?

File: package-lock.json (hunk 2 above)
Comment (pablodgonzalez) [inline package-lock.json]: No problem

File: webpack.config.js (hunk 3 above)
Comment (pablodgonzalez) [inline webpack.config.js]: when I run "npm run build" with './dist/' I get:
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.output.path: The provided value "./dist/" is not an absolute path!
I can undo the change but I can't build neither test the library

**Comment by pablodgonzalez** [comment]
I've made this component for angular https://github.com/senzil/angular-datetime-range. So let me know your opinion.
Cheers!

File: webpack.config.js (hunk 3 above)
Comment (gf3) [inline webpack.config.js]: sounds like the version of webpack you're using may not be the same as the [one used in development](https://github.com/rotaready/moment-range/blob/master/yarn.lock#L4143-L4145)

File: CHANGELOG.md (hunk 1 above)
Comment (Rhysjc) [inline CHANGELOG.md]: Nope, all changes to this file can be removed since we'll sort this ourselves

File: webpack.config.js (hunk 3 above)
Comment (Rhysjc) [inline webpack.config.js]: We'd really like to merge this in if you can revert this change

**Comment by Rhysjc** [comment]
@pablodgonzalez That component looks cool, nice work! If you could make the changes listed in my review we can get this merged in

**Comment by pablodgonzalez** [comment]
@Rhysjc All seems good now. Check it out.

File: README.md [hunk 4]
```diff
@@ -21,6 +21,7 @@ Fancy date ranges for [Moment.js][moment].
     - [Within](#within)
     - [Overlaps](#overlaps)
     - [Intersect](#intersect)
+    - [IsRange](#IsRange)
```
Comment (TristanJM) [inline README.md]: Change `#IsRange` to lowercase (so doc linking works correctly)

File: README.md [hunk 5]
```diff
@@ -258,6 +259,15 @@ What are the intersecting ranges?
 range.intersect(range2); // [moment.range(lol, end)]
 ```
 
+#### IsRange
+
+Is that a Range?
```
Comment (TristanJM) [inline README.md]: Change to `Is it a range?`

**Comment by pablodgonzalez** [comment]
@TristanJM It's Done!

**Comment by webdobe** [comment]
Is this in yet?
